emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -172,18 +174,22 @@ public void toggle() {
 
       if (isKeyboardOpen) {
         // If the keyboard is visible, simply show the emoji popup.
-        showAtBottom();
+        editText.postDelayed(new Runnable() {
+          @Override public void run() {
+            showAtBottom();
+          }
+        }, 50);
```

Comment:
Because in rare cases, due to race conditions, we get a smaller height from the keyboard than expected (because it's not fully shown), so the emoji popvuiew is smaller than needed.